### PR TITLE
Add some notes about configuring Json formatters

### PR DESCRIPTION
**Pull request by pranavkm**
Fixes https://github.com/aspnet/AspNetCore.Docs/issues/241

### CODE REVIEW

**Comment by pranavkm** [comment]
I'm intentionally leaving our `IJsonHelper`. We don't have a good section to put it in, but more importantly given how infrequently we hear of it, it seems like a fairly low use API.

File: aspnetcore/web-api/advanced/formatting.md [hunk 1]
```diff
@@ -100,21 +100,38 @@ If your application needs to support additional formats beyond the default of JS
...
-    options.SerializerOptions.WriterSettings.Indented = true;
+    // Use the default property (Pascal) casing.
+    options.SerializerOptions.PropertyNamingPolicy = null;
+
+    // Configure a custom converter.
+    options.SerializerOptions.Converters.Add(new MyCustomJsonConverter());
 });
 ```
 
+Output serialization options on a per-action basis can be configured using `JsonResult`. For example,
```
Comment (scottaddie) [inline aspnetcore/web-api/advanced/formatting.md]:
```suggestion
Output serialization options, on a per-action basis, can be configured using `JsonResult`. For example:
```

File: aspnetcore/web-api/advanced/formatting.md [hunk 2]
```diff
@@ -125,6 +142,32 @@ release. Continue using the `Newtonsoft.Json`-based formatters if your ASP.NET C
...
+{
+    // Use the default property (Pascal) casing
+    options.SerializerSettings.ContractResolver = new DefautlContractResolver();
+
+    // Configure a custom converter
+    options.SerializerOptions.Converters.Add(new MyCustomJsonConverter());
+});
+```
+
+Output serialization options on a per-action basis can be configured using `JsonResult`. For example,
```
Comment (scottaddie) [inline aspnetcore/web-api/advanced/formatting.md]:
```suggestion
Output serialization options, on a per-action basis, can be configured using `JsonResult`. For example:
```

**Comment by pranavkm** [comment]
🆙 📅

**Comment by scottaddie** [comment]
Thank you for the PR, @pranavkm!

**Pull request merged**